src/collections/_documentation/workflow/releases.md

```diff
@@ -35,36 +35,40 @@ After this, you should see information about the release, such as new issues and
 
 {% asset releases-overview.png %}
 
-### Link a Repository and Associate Commits {#link-repository}
+### Create Release and Associate Commits {#create-release}
 
-In this step you tell Sentry which commits are associated with a release, allowing Sentry to pinpoint which commits likely caused an issue, and allowing you to resolve an issue by including the issue number in your commit message.
+In this step you tell Sentry about a new release and which commits are associated with it. This allows Sentry to pinpoint which commits likely caused an issue, and allow your team to resolve issues by referencing the issue number in a commit message.
 
-This can be done either with an integration or without one. 
+There are two ways to create a release and associate commits: using a repository integration (recommended), or by manually supplying Sentry with your own commit metadata.
 
-####  Link a Repository with Integrations
+#### Link a Repository
 
-This is a 2-step process:
+Using one of Sentry's repository integrations (e.g. GitHub, GitLab, Bitbucket, etc.) is the easiest way to connect your commit metadata to Sentry. For a list of available integrations, go to Organization Settings > Integrations.
 
-##### a. Link a Repository via Integrations
+Note that you'll need to be an Owner or Manager of your Sentry organization to set up or configure an integration. Read more about roles in Sentry [here]({%- link _documentation/accounts/membership.md -%}).
```

Comment:
Hi Ben, this Note can go in an alert box with the level = warning.

Reply:
Done!